app/models/conditions_response/shf_app_no_uploaded_files_alert.rb

```diff
@@ -19,11 +19,11 @@ def send_alert_this_day?(timing, config, user)
 
       # If the applicant has said they will email or (postal) mail the files,
       # don't send this alert.
-      if shf_app.file_delivery_method.email? || shf_app.file_delivery_method.mail?
+
+      if shf_app&.file_delivery_method&.email? || shf_app&.file_delivery_method&.mail?
```

Comment:
I don't think we need the first safe-nav operator (`shf_app&.`) since line 12 provides a guard against `shf_app == nil`.

Reply:
exactly right.  Good catch!